<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="JOHAN-NB" failures="1" tests="3" name="TimeOut Program" time="0.224" errors="0" timestamp="2024-09-08T11:13:02 WIB">
  <testcase classname="com.javatpoint.annotations.Time_out" name="testcase1" time="0.213">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.javatpoint.annotations.Time_out.testcase1() didn&amp;apos;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.javatpoint.annotations.Time_out.testcase1() didn't finish within the time-out 200
at java.base/java.lang.Thread.sleepNanos0(Native Method)
at java.base/java.lang.Thread.sleepNanos(Thread.java:491)
at java.base/java.lang.Thread.sleep(Thread.java:522)
at com.javatpoint.annotations.Time_out.testcase1(Time_out.java:9)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="com.javatpoint.annotations.Time_out" name="testcase2" time="0.001"/>
  <testcase classname="com.javatpoint.annotations.Time_out" name="testcase3" time="0.001"/>
</testsuite> <!-- TimeOut Program -->
